src/app/static/src/app/components/figures/graphs/longFormatDataSeries.ts

```diff
@@ -31,7 +31,11 @@ export function useLongFormatData(props: Props) {
                 }
 
                 if (error_col && error_col_minus) {
-                    error_array.push(getErrorInterval(row[error_col_minus], row[meta.x_col], row[error_col]));
+                    error_array.push(getErrorInterval(
```

Comment:
Was this an unrelated bug where values weren't being evaluated? ll the changes in mintr relate to error_y columns.

Reply:
`error_x` [here](https://github.com/mrc-ide/mintr/pull/53/files#diff-7cc1b802ddf785efab159ad3514e9fbb10ee16a104dde60c4765fa8cf4e033bd) (but apologies if I've misunderstood!)

Reply:
Apologies, yes, I completely misread that. But this means we are clipping the cases averted values to 0 in the costs vs strategy graph, but not clipping them on the impact tab, doesn't it? That seems inconsistent, is it desired behaviour?

Reply:
It does... and something I'm looking for feedback on from Ellie. She suggested that we avoid negative costs - which are arguably nonsensical whereas negative cases less so. Clipping everything at zero is certainly a lot easier (a one-line change to error interval code) but I'm not suggesting we commit to either approach without further guidance - especially if the numbers themselves need to be revised.